package.json

```diff
@@ -14,11 +14,11 @@
   "author": "Blackbaud",
   "license": "MIT",
   "dependencies": {
-    "@blackbaud/skyux": "2.0.0-beta.26",
+    "@blackbaud/skyux": "2.0.0-beta.29",
```

Comment:
Whichever version we bump skyux2 and builder to, needs to be updated on stache as well:
https://github.com/blackbaud/stache2/pull/182